client/src/Components/Layouts/Captains/EditCaptain/editCaptain.jsx

```diff
@@ -0,0 +1,301 @@
+import React from "react";
+import { Button, Modal, Form, Select, Input } from "antd";
+import { Upload, Icon } from 'antd';
+import { notification } from 'antd';
+import './style.css';
+
+const { Option } = Select;
+
+const CollectionCreateForm = Form.create({ name: "form_in_modal" })(
+  class extends React.Component {
+    state = {
+      loading: false,
+      imageUrl: null,
+      photoUrl: null,
+
+    };
+
+
+    handleReuest = e => {
+      this.setState({ imageUrl: e.file });
+    };
+    componentDidMount() {
+      const id = 6;
+      // const { id } = this.props.id;
+      fetch(`/api/v1/getCaptainDetails/${id}`)
+        .then(res => res.json())
+        .then(res => {
+          if (res.result) {
+            const rows = res.result[0];
+            const status = rows.b_status ? "true" : "false";
+            this.props.form.setFieldsValue({
+              name: rows.s_name, email: rows.s_email, phone: rows.s_mobile_number,
+              address: rows.s_address, IDNumber: rows.s_id_number, licenceNumber: rows.s_driver_licence_number,
+              status, file: rows.s_image,
+            });
+            var formData = this.props.form.getFieldsValue();
+            fetch(`/api/v1/image/${rows.s_image}`)
+              .then(res => res.arrayBuffer())
+              .then(response => {
+                let typeArray = new Uint8Array(response);
+                const stringChar = String.fromCharCode.apply(null, typeArray);
+
+                this.setState({
+                  photoUrl: stringChar
```

Comment:
@naremanhilles is stringChar a dataUrl?
I wonder if it would work if you directly set the photoUrl to `/api/v1/image/${rows.s_image}`?